README.md

```diff
@@ -1,7 +1,12 @@
 # parliament-client-template
 
 ## 🚀 Quick start
+### Prerequisites
+1. `node` 14.15 <= version < 16 (e.g., if using nvm, type `nvm use 14.17.5`)
```

Comment:
Maybe this is just me but it took me a while to kinda fully understand this 😂 would it be too much if we just spelled everything out? ie:

```suggestion
1. `node`: Use a version between 14.15 and 16 (e.g., if using nvm, type `nvm use 14.17.5`)
```